readme.md

```diff
@@ -1,6 +1,8 @@
 > This is a list of must-watch videos devoted to JavaScript
 
 ## 2015
+1. [Andre Staltz: What if the user was a function?](https://www.youtube.com/watch?v=1zj7M1LnJV4) [32:19]
+1. [Sufian Rhazi: Transform your codebase](https://www.youtube.com/watch?v=X8WgkMbEAVQ) [33:49]
```

Comment:
This one, unfortunately, is not a must see talk from my point of view. It's indeed a good one and very useful. However, it's not really novel. The whole talk is about combination of known techniques and tools like AST, Esprima, Feature toggles, etc. If people know them separately, they can come up w/ the same solution.